main/main.cpp

```diff
@@ -344,6 +344,8 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  --resolution <W>x<H>                         Request window resolution.\n");
 	OS::get_singleton()->print("  --position <X>,<Y>                           Request window position.\n");
 	OS::get_singleton()->print("  --single-window                              Use a single window (no separate subwindows).\n");
+	OS::get_singleton()->print("  -xr                                          Turn XR mode on (same as --xr-mode on).\n");
```

Comment:
What's the use case for `-xr` since it's a subset of the `--xr-mode` option?

Reply:
Just a shortcut, I'm not sure if we should keep it.

Reply:
ok changed this so `-xr` and `--xr-mode` are basically the same thing.